Autopilot/Src/imu.cpp

```diff
@@ -45,7 +45,7 @@
 #define MAG_IF_3_DATA_MODE 0x02
 #define MAG_IF_2_DATA_MODE 0x4C
 #define MAG_IF_1_DATA_MODE 0x42
-#define MAG_REFRESH_50_HZ 0x07  // TODO mag should refresh at 200 Hz as well, but I cant seem to get that to work.
```

Comment:
Nice catch, but perhaps not best to include with this PR since it is concerning airspeed. You could ask Dhruv U to add it in since his PR concerns itself with IMU.

Reply:
ngl tho, after competition when all the work is done we should take a few days to just clean up everything. Then everyone can rebase and have a clean env. to start new tasks on

Reply:
I don't remember adding this. and have no clue what it does. So I'm going to undo the change.

Reply:
Oh, you're just fixing a typo in the comment